front/pages/api/w/[wId]/assistant/agent_configurations/[aId]/scope.ts

```diff
@@ -86,17 +90,63 @@ async function handler(
           },
         });
       }
+
       if (
         assistant.scope !== "private" &&
         bodyValidation.right.scope === "private"
       ) {
-        return apiError(req, res, {
-          status_code: 400,
-          api_error: {
-            type: "invalid_request_error",
-            message: "Non-private assistants cannot be set back to private.",
-          },
+        // switching an assistant back to private: removing it from all user's
```

Comment:
all users' list or all the user's list?

Reply:
indeed, `all users' lists` was the right wording